src/style.css

```diff
@@ -2,0 +1,29 @@
+body{
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  box-sizing: border-box;
+  background-repeat: no-repeat;
+  height: 16px;
+  width: 16px;
+  background-position: center;
+  background-image: url('./images/star.svg');
+}
```

Comment:
It's a good practice to provide alternative images in case the main one doesn't work or takes time to load. You can use the CSS `background` property to set multiple background images. If the first background image is not found, the next one will be used.